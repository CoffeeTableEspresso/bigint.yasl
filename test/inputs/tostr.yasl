const bigint = require('bigint')

echo bigint('54321')->tostr('r')
echo bigint('12345')->tostr('b')
echo bigint('9876')->tostr('d')
echo bigint('121212')->tostr('x')

