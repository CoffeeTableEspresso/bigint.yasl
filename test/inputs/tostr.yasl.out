bigint('54321')
0b11000000111001
9876
0x1D97C

